tests/Issue2469.py

```diff
@@ -0,0 +1,41 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2023 Mauricio Martinez <mauricio.martinez@premise.com>             #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+from . import Framework
+
+
+class Issue2469(Framework.TestCase):  # https://github.com/PyGithub/PyGithub/issues/2469
```

Comment:
```suggestion
class Issue2469(Framework.TestCase):  # https://github.com/PyGithub/PyGithub/issues/2469
    tokenAuthMode = True

```

This will fix ci.